src/scss/variables.scss

```diff
@@ -16,6 +16,7 @@ $content-padding-bottom: 2.5rem;
 $content-padding-left: 5rem;
 $content-padding-right: 6.5rem;
 $content-side-paddings-sm: 2rem;
+$list-item-margin: 2rem;
```

Comment:
you don't use it outside the "list" component. place it there

Reply:
also it's a gap, not margin

Reply:
removed this variable, and assigned value to grid-gap basic way